migrations/909-website-reviewers.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO groups (id, name, rules, created, modified, notes)
+VALUES (50080, 'Website Reviewers', 'Websites:Review', NOW(), NOW(), 'Users who can review websites');
```

Comment:
Let's not include IDs in our migrations in the future; this caused a deploy error for duplicate IDs.